lang/en.json

```diff
@@ -1583,6 +1583,34 @@
     "written": "Written",
     "yard": "yard",
     "yards": "yards",
-    "youDoNotHavePermssion": "You do not have permission to do that."
+    "youDoNotHavePermssion": "You do not have permission to do that.",
+    "Dialog": {
+      "Save": "Save",
+      "Overwrite": "Overwrite"
+    },
+    "Importer": {
+      "Prompt": {
+        "Title": "Import Character Data for {name}",
+        "Description": "Select a file exported from GCS 5+.",
```

Comment:
We no longer require the file to be exported from GCS; this should probably read something like "Select a GCS 5+ character file".